x/docs/md/intro/getting-started.md

```diff
@@ -14,3 +14,11 @@ Once you have submitted your first exercise, there are several other things you
 1. Browse other solutions to the same problem.
    Provide feedback: ask questions, suggest improvements, and ponder style and readability.
 1. Fetch a new exercise and solve it.
+
+## Command Line Tips 
+
+You can submit a header file along with your implementation using:  
```

Comment:
How about making it generic, and then adding a specific tip about header files (because the `{a,b}` syntax is really nice).